index.html

```diff
@@ -149,19 +146,20 @@
                             or sign-ups required.</p>
                         <ul class="list-style-type-none d-flex lang weight--500">
                             <li>html</li>
+                            <li>Ruby on rails</li>
                             <li>css</li>
                             <li>javaScript</li>
                         </ul>
                         <button class="btn weight--500">See Project</button>
                     </aside>
                 </article>
             </div>
-        </section>
+        </main>
 
-        <footer>
-            <section class="container">
-                <article>
-                    <aside class="pt-114">
+        <footer class="pt-114">
+            <section class="container about-me">
+                <article class="flex-basis-5">
+                    <aside>
```

Comment:
♻️ The desktop version of the about section looks really nice. There is still a thing you should take care of. Currently, when surpassing the required breakpoint (768px) this section is not changing as it should. It only becomes desktop friendly at 1131px

![image](https://user-images.githubusercontent.com/26911718/206576687-97fcd1c9-b1b0-4acc-878d-56a220d857c7.png)


- [ ] Please make sure the desktop styles are applied to all the sections on the page as per the provided template. Being consistent throughout the page would make it way more professional.